packages/codefixes/src/codefixes.ts

```diff
@@ -1,26 +1,54 @@
+import { applyTextChange, normalizeTextChanges } from '@rehearsal/ts-utils';
 import { BaseCodeFixCollection } from './base-codefix-collection.js';
 import { CodeFixesProvider } from './codefixes-provider.js';
 import { TypescriptCodeFixCollection } from './typescript-codefix-collection.js';
+import { Diagnostics } from './diagnosticInformationMap.generated.js';
 
 import { AddErrorTypeGuardCodeFix } from './fixes/addErrorTypeGuard/index.js';
 import { AddMissingExportCodeFix } from './fixes/addMissingExport/index.js';
 import { MakeMemberOptionalCodeFix } from './fixes/makeMemberOptional/index.js';
 import { AddMissingTypesBasedOnInheritanceCodeFix } from './fixes/addMissingTypesBasedOnInheritance/index.js';
 import { AddMissingTypesBasedOnInlayHintsCodeFix } from './fixes/addMissingTypesBasedOnInlayHints/index.js';
+import type { CodeFixAction } from 'typescript';
 
 export const codefixes = new CodeFixesProvider([
   new BaseCodeFixCollection({
-    7050: new AddMissingTypesBasedOnInheritanceCodeFix(),
+    [Diagnostics._0_implicitly_has_an_1_return_type_but_a_better_type_may_be_inferred_from_usage
+      .code]: new AddMissingTypesBasedOnInheritanceCodeFix(),
   }),
   new BaseCodeFixCollection({
-    18046: new AddErrorTypeGuardCodeFix(),
-    2571: new AddErrorTypeGuardCodeFix(),
-    2790: new MakeMemberOptionalCodeFix(),
-    4082: new AddMissingExportCodeFix(),
-    7050: new AddMissingTypesBasedOnInlayHintsCodeFix(),
+    [Diagnostics._0_is_of_type_unknown.code]: new AddErrorTypeGuardCodeFix(),
+    [Diagnostics.Object_is_of_type_unknown.code]: new AddErrorTypeGuardCodeFix(),
+    [Diagnostics.The_operand_of_a_delete_operator_must_be_optional.code]:
+      new MakeMemberOptionalCodeFix(),
+    [Diagnostics.Default_export_of_the_module_has_or_is_using_private_name_0.code]:
+      new AddMissingExportCodeFix(),
+    [Diagnostics._0_implicitly_has_an_1_return_type_but_a_better_type_may_be_inferred_from_usage
```

Comment:
I think `[7050]: [new AddMissingTypesBasedOnInlayHintsCodeFix()]` is easy to work with. 

You can easily overview which error code is associated with a particular fix, you don't need to read through messages or do additional "decoding" steps to recognize when an error code is associated.